Installation/installation.json

```diff
@@ -1,3 +1,68 @@
 {
-    "installationService": "CommonGateway\\KlantenBundle\\Service\\InstallationService"
+    "installationService": "CommonGateway\\KlantenBundle\\Service\\InstallationService",
+    "collections": [{
+        "reference": "https://vng.opencatalogi.nl/collection/Klanten.collection.json",
+        "schemaPrefix": "https://vng.opencatalogi.nl/schemas/brc"
```

Comment:
https://klantenBundle.commonground.nu/klant
Zie installationService->createCollections()

Reply:
Like this, right? Kinda looked over it.